starter-project-web-vue2/components/yared/Blog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+<div class="blog">
+      <v-card
+    :to="'/yared/blogs/' + blog._id"
+    class="mx-auto"
```

Comment:
Good indentation of a code can increase readability. So use lint or other tools to fix it.